functorch/_src/aot_autograd.py

```diff
@@ -409,14 +417,21 @@ def forward(ctx, *deduped_flat_tensor_args):
                 CompiledFunction.compiled_fw, deduped_flat_tensor_args
             )
             num_outs = CompiledFunction.num_outs
-            ctx.save_for_backward(*fw_outs[num_outs:])
+            num_symints = CompiledFunction.num_symints
+            if num_symints > 0:
+                ctx.save_for_backward(*fw_outs[num_outs:-num_symints])
+                ctx.symints = fw_outs[-num_symints:]
+            else:
+                ctx.save_for_backward(*fw_outs[num_outs:])
+                ctx.symints = []
+
             return tuple(fw_outs[0:num_outs])
 
         @staticmethod
         @disable_torchdynamo
         def backward(ctx, *flat_args):
-            contiguous_args = [t.contiguous() for t in flat_args]
-            all_args = list(ctx.saved_tensors) + list(contiguous_args)
+            contiguous_args = [t.contiguous() if torch.is_tensor(t) else t for t in flat_args]
+            all_args = list(ctx.symints) + list(ctx.saved_tensors) + list(contiguous_args)
```

Comment:
@bdhirsh this looks like this would fix the None problem

Reply:
it did :)